docs/README.md

```diff
@@ -12,3 +12,177 @@ We currently have the following components available for React applications:
 -   [Score Card](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/ScoreCard.md)
 -   [Spacer](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/Spacer.md)
 -   [User Menu](https://github.com/brightlayer-ui/react-component-library/blob/dev/docs/UserMenu.md)
+
+---
+
+## Style Overrides Guide
+
+Material UI, and by extension, Brightlayer UI provides several different ways to customize a component's styles. Your specific context will determine which one is ideal. From narrowest to broadest use case, here are the options:
+
+### 1. The `sx` prop
+
+The [sx prop](https://mui.com/system/basics/#the-sx-prop) is the best option for adding style overrides to a single instance of a component in most cases.
+
+#### Usage
+
+```tsx
+import { InfoListItem } from '@brightlayer-ui/react-components';
+...
+<InfoListItem
+    title={'Info List Item'}
+    sx={{ my: 2, backgroundColor: '#efefef' }}
+/>
+```
+
+### 2. Overriding nested component styles
+
+To customize a specific part of a component, you can use the global CSS class provided by Brightlayer UI inside the sx prop. These global CSS class names can be found in the `Style Overrides` section of the documentation for each component. As an example, let's say you want to customize the InfoListItem component's icon size and padding.
+
+#### Usage
+
+```tsx
+import { InfoListItem } from '@brightlayer-ui/react-components';
+import { Leaf } from '@brightlayer-ui/icons-mui';
+...
+<InfoListItem
+    title={'Info List Item'}
+    icon={<Leaf />}
+    sx={{ width: '100%', '& .BluiInfoListItem-icon' : { ml: '2px', fontSize: '1rem' } }}
+/>
+```
+
+### 3. Overriding styles with class names
+
+If you want to override a component's styles using custom classes, you can use the className prop, available on each component. To override the styles of a specific part of the component, use the global CSS classes provided, as described in the previous section "Overriding nested component styles".
```

Comment:
This `className` description, second sentence, seems to be referring to the instructions for sx overrides.  Perhaps an example would be helpful here.